### ISSUE TITLE
:comet: Epic 1: Move Pals to Knapsack

### ISSUE THREAD

**Issue by jillpe** [scientist-softserv/palni_palci_knapsack#2]
## Goal

To have a near feature parity of current Pals Hyku instance and a knapsack based instance.

blocked by: 
- https://github.com/scientist-softserv/hykuup_knapsack/issues/188
- https://github.com/scientist-softserv/hykuup_knapsack/issues/190

## Prerequisite Task
- [ ] Update knapsack with Hyku Knapsack Prime
  - https://github.com/scientist-softserv/hykuup_knapsack/issues/191

- [x] #11

### Strategies

http://playbook-staging.notch8.com/en/samvera/hyku/knapsack/converting-to

- [ ] - Leverage Adventist's work on converting their Hyku to Knapsack
- [ ]   - https://github.com/scientist-softserv/adventist_knapsack/issues/112
- [ ]  https://github.com/scientist-softserv/palni_palci_knapsack/issues/10
- [ ] - Address [diff](https://github.com/scientist-softserv/palni-palci/issues/970) between Pals Hyku and Hyku prime
- [ ] - Move Pals specific functionality into their Knapsack
- [ ] - Move "contribute back" functionality into Hyku Prime
- [ ]   - Consider that this can either be done as "move into Knapsack" (to meet time requirements) then "move into Hyku Prime"

<details><summary>Source Control Strategies</summary>

#### Overview

We are dealing with:

- Knapsacks
  - https://github.com/scientist-softserv/palni_palci_knapsack/
  - https://github.com/scientist-softserv/adventist_knapsack/
  - http://github.com/samvera-labs/hyku_knapsack
- Hykus
  - http://github.com/scientist-softserv/palni-palci
    - Hyrax v3.6.0 (via `privkey_for_ga4_3.x`)
  - http://github.com/samvera/hyku (Hyku Prime)
    - Hyrax v3.5.0 (via `hyrax-v3.5.0`)
  - http://github.com/scientist-softserv/adventist-dl; there is code in here that is not in Adventist Knapsack
    - Hyrax v2.9.6

#### Managing This All

Alas, this is TBD.
</details>

## Tasks
- [X] Initial Analysis.  What files need to go where?
- [X] Routing Files to Location. Copy files to correct location (Hyku Prime, Knapsack Prime, or PALS's knapsack)
- [X] Hyku Prime rubocop passing with latest files
- [X] Hyku Prime specs passing with latest files
- [ ] Mint Hyku Prime release candidate
- [ ] Update PALs Knapsack to use latest release candidate
- [ ] PALs Knapsack rubocop passing with latest files
- [ ] PALs Knapsack specs passing with latest files
- [ ] PALs Knapsack integration testing (e.g. Hyku test plan)
- [ ] Client QA  
- [ ] Client Signoff

**Comment by ShanaLMoore** [comment]
blocked until Pals contributions have been merged into hyku main

**Comment by ShanaLMoore** [comment]
will need to reinstate this somehow: it was causing the pipeline and deploys to fail for hyku



![Image](https://github.com/scientist-softserv/palni_palci_knapsack/assets/10081604/5734bbab-896e-4c7a-a6d6-a7bec7c47e09)

**Comment by ShanaLMoore** [comment]
Finished in 12 minutes 55 seconds (files took 14.37 seconds to load)
1981 examples, 211 failures, 42 pending

Failed examples:

- [ ] rspec ./spec/services/hyrax/accessibility_features_service_spec.rb:25 # Hyrax::AccessibilityFeaturesService.microdata_type when id is in the i18n gives schema.org type
- [ ] rspec ./spec/indexers/cdl_indexer_spec.rb:16 # CdlIndexer#generate_solr_document when indexing contributing_library indexes the label value into contributing_library_sim
- [ ] rspec ./spec/indexers/cdl_indexer_spec.rb:12 # CdlIndexer#generate_solr_document when indexing contributing_library indexes the term value into contributing_library_tesim
- [ ] rspec ./spec/controllers/hyrax/cdls_controller_spec.rb:14 # Hyrax::CdlsController includes Hyrax::IiifAv::ControllerBehavior
- [ ] rspec ./spec/controllers/hyrax/cdls_controller_spec.rb:27 # Hyrax::CdlsController#presenter initializes a presenter
- [ ] rspec ./spec/parsers/bulkrax/oer_csv_parser_spec.rb:57 # Bulkrax::OerCsvParser#records_with_related_items returns a hash of identifiers with related items
- [ ] rspec ./spec/parsers/bulkrax/oer_csv_parser_spec.rb:77 # Bulkrax::OerCsvParser#create_alternate_version_relationships invokes Bulkrax::RelatedMembershipsJob
- [ ] rspec ./spec/parsers/bulkrax/oer_csv_parser_spec.rb:37 # Bulkrax::OerCsvParser#records_with_newer_versions returns a hash of identifiers with newer versions
- [ ] rspec ./spec/parsers/bulkrax/oer_csv_parser_spec.rb:84 # Bulkrax::OerCsvParser#create_related_item_relationships invokes Bulkrax::RelatedMembershipsJob
- [ ] rspec ./spec/parsers/bulkrax/oer_csv_parser_spec.rb:63 # Bulkrax::OerCsvParser#create_previous_version_relationships invokes Bulkrax::RelatedMembershipsJob
- [ ] rspec ./spec/parsers/bulkrax/oer_csv_parser_spec.rb:43 # Bulkrax::OerCsvParser#records_with_alternate_versions returns a hash of identifiers with alternate versions
- [ ] rspec ./spec/parsers/bulkrax/oer_csv_parser_spec.rb:50 # Bulkrax::OerCsvParser#records_with_alternate_versions with complex relationships parses complex related items appropriately
- [ ] rspec ./spec/parsers/bulkrax/oer_csv_parser_spec.rb:70 # Bulkrax::OerCsvParser#create_newer_version_relationships invokes Bulkrax::RelatedMembershipsJob
- [ ] rspec ./spec/parsers/bulkrax/oer_csv_parser_spec.rb:23 # Bulkrax::OerCsvParser#records_with_previous_versions returns a hash of identifiers with previous versions
- [ ] rspec ./spec/parsers/bulkrax/oer_csv_parser_spec.rb:30 # Bulkrax::OerCsvParser#records_with_previous_versions with complex relationships parses complex related items appropriately
- [ ] rspec ./spec/hyku_specs/features/admin_dashboard_spec.rb:58 # Admin Dashboard as an administrator displays the add-users-to-groups page without the hidden form field
- [ ] rspec ./spec/hyku_specs/features/admin_dashboard_spec.rb:50 # Admin Dashboard as an administrator shows the status page
- [ ] rspec ./spec/features/etd_show_spec.rb:24 # Etd show page attribute labels displays the custom Etd labels
- [ ] rspec ./spec/hyku_specs/controllers/splash_controller_spec.rb:5 # SplashController get index is successful
- [ ] rspec ./spec/hyku_specs/views/account_sign_up/new.html.erb_spec.rb:10 # account_sign_up/new renders new account form
- [ ] rspec ./spec/parsers/bulkrax/csv_parser_decorator_spec.rb:203 # Bulkrax::CsvParserDecorator#valid_import? when the csv headers do not have underscores still returns true
- [ ] rspec ./spec/parsers/bulkrax/csv_parser_decorator_spec.rb:214 # Bulkrax::CsvParserDecorator#valid_import? when the csv header is the parser_mappings value still returns true
- [ ] rspec ./spec/features/application_config_spec.rb:14 # Application Configuration ApplicationController#view_paths first element is expected to eq "/app/samvera/hyrax-webapp/app/views"
- [ ] rspec ./spec/hyku_specs/routing/curation_concerns_spec.rb:4 # /concern/generic_works routing routes to manifest
- [ ] rspec ./spec/hyku_specs/features/oai_pmh_spec.rb:20 # OAI PMH Support oai interface with works present lists metadata prefixes
- [ ] rspec './spec/hyku_specs/features/oai_pmh_spec.rb[1:1:3:1]' # OAI PMH Support oai interface with works present with the oai_hyku prefix retrieves a list of records
- [ ] rspec './spec/hyku_specs/features/oai_pmh_spec.rb[1:1:3:2]' # OAI PMH Support oai interface with works present with the oai_hyku prefix retrieves a single record
- [ ] rspec './spec/hyku_specs/features/oai_pmh_spec.rb[1:1:3:3]' # OAI PMH Support oai interface with works present with the oai_hyku prefix retrieves a list of identifiers
- [ ] rspec './spec/hyku_specs/features/oai_pmh_spec.rb[1:1:2:1]' # OAI PMH Support oai interface with works present with the oai_dc prefix retrieves a list of records
- [ ] rspec './spec/hyku_specs/features/oai_pmh_spec.rb[1:1:2:2]' # OAI PMH Support oai interface with works present with the oai_dc prefix retrieves a single record
- [ ] rspec './spec/hyku_specs/features/oai_pmh_spec.rb[1:1:2:3]' # OAI PMH Support oai interface with works present with the oai_dc prefix retrieves a list of identifiers
- [ ] rspec ./spec/hyku_specs/features/oai_pmh_spec.rb:52 # OAI PMH Support when using the oai_hyku prefix includes non-DC fields
- [ ] rspec ./spec/models/work_authorization_spec.rb:73 # WorkAuthorization.handle_signin_for! when given a work_pid will re-authorize the given work and expire non-specified works
- [ ] rspec ./spec/models/work_authorization_spec.rb:61 # WorkAuthorization.handle_signin_for! when given a work_pid and a scope will authorize the given work_pid and scope’s work
- [ ] rspec ./spec/models/work_authorization_spec.rb:86 # WorkAuthorization.handle_signin_for! when not given a work_pid will de-authorize all authorizations that have expired but not those that have not expired
- [ ] rspec ./spec/models/work_authorization_spec.rb:102 # WorkAuthorization.authorize! gives the borrowing user the ability to "read" the work
- [ ] rspec ./spec/models/work_authorization_spec.rb:120 # WorkAuthorization.revoke! gracefully handles revoking that which was never authorized
- [ ] rspec ./spec/models/work_authorization_spec.rb:112 # WorkAuthorization.revoke! revokes an authorized user from being able to "read" the work
- [ ] rspec ./spec/hyku_specs/config/application_spec.rb:15 # Hyku::Application.user_devise_parameters is expected to eq [:database_authenticatable, :invitable, :registerable, :recoverable, :rememberable, :trackable, :validatable, :omniauthable, {:omniauth_providers=>[:saml, :openid_connect, :cas]}]
- [ ] rspec ./spec/features/roles_spec.rb:20 # Site Roles as an administrator updates user roles
- [ ] rspec ./spec/features/roles_spec.rb:13 # Site Roles as an administrator lists user roles
- [ ] rspec './spec/hyku_specs/requests/work_editor_role_spec.rb[1:1:2:1]' # Work Editor role read permissions with authenticated visibility can see the show page for works it deposited
- [ ] rspec './spec/hyku_specs/requests/work_editor_role_spec.rb[1:1:1:1]' # Work Editor role read permissions with open visibility can see the show page for works it deposited
- [ ] rspec './spec/hyku_specs/requests/work_editor_role_spec.rb[1:1:3:1]' # Work Editor role read permissions with restricted visibility can see the show page for works it deposited
- [ ] rspec ./spec/hyku_specs/indexers/image_resource_indexer_spec.rb:12 # ImageResourceIndexer behaves like a Hyrax::Resource indexer #to_solr indexes base resource fields
- [ ] rspec ./spec/hyku_specs/controllers/catalog_controller_spec.rb:21 # CatalogController GET /show without access is redirects to sign in
- [ ] rspec ./spec/hyku_specs/controllers/catalog_controller_spec.rb:13 # CatalogController GET /show with access is successful
- [ ] rspec ./spec/jobs/create_group_and_add_members_job_spec.rb:50 # CreateGroupAndAddMembersJob#perform when has all its child works assigns the group to the read_groups of each work's member's member
- [ ] rspec ./spec/jobs/create_group_and_add_members_job_spec.rb:37 # CreateGroupAndAddMembersJob#perform when has all its child works assigns the group to the work read_groups
- [ ] rspec ./spec/jobs/create_group_and_add_members_job_spec.rb:33 # CreateGroupAndAddMembersJob#perform when has all its child works creates a group
- [ ] rspec ./spec/jobs/create_group_and_add_members_job_spec.rb:43 # CreateGroupAndAddMembersJob#perform when has all its child works assigns the group to the read_groups of each work's member
- [ ] rspec ./spec/jobs/create_group_and_add_members_job_spec.rb:64 # CreateGroupAndAddMembersJob#perform when does not have all its child works retries the job
- [ ] rspec ./spec/jobs/create_group_and_add_members_job_spec.rb:70 # CreateGroupAndAddMembersJob#perform when does not have all its child works does not retry the job more than 5 times
- [ ] rspec ./spec/hyku_specs/views/proprietor/accounts/edit.html.erb_spec.rb:12 # proprietor/accounts/edit with connections renders the edit account form
- [ ] rspec ./spec/hyku_specs/views/proprietor/accounts/edit.html.erb_spec.rb:23 # proprietor/accounts/edit without connections renders the edit account form
- [ ] rspec ./spec/hyku_specs/models/datacite_endpoint_spec.rb:6 # DataCiteEndpoint 
- [ ] rspec ./spec/hyku_specs/models/generic_work_spec.rb:16 # GenericWork class configuration metadata is expected to have property :bulkrax_identifier
- [ ] rspec ./spec/models/concerns/account_settings_spec.rb:39 # AccountSettings#public_settings when is_superadmin is false returns all settings except private, disabled, and superadmin settings
- [ ] rspec ./spec/hyku_specs/models/hyrax/contact_form_spec.rb:21 # Hyrax::ContactForm headers no email set uses the hyrax setting
- [ ] rspec ./spec/helpers/blacklight/advanced_search_helper_spec.rb:13 # Blacklight::AdvancedSearchHelper#primary_search_fields returns the client specified advanced search fields
- [ ] rspec ./spec/jobs/destroy_split_pages_job_spec.rb:15 # DestroySplitPagesJob#perform destroys the child works when the work is destroyed
- [ ] rspec ./spec/services/hyrax/accessibility_hazards_service_spec.rb:25 # Hyrax::AccessibilityHazardsService.microdata_type when id is in the i18n gives schema.org type
- [ ] rspec ./spec/hyku_specs/controllers/admin/groups_controller_spec.rb:8 # Admin::GroupsController as an anonymous user GET #index 
- [ ] rspec ./spec/hyku_specs/controllers/admin/groups_controller_spec.rb:51 # Admin::GroupsController as an admin user with an existing group PATCH #update updates attribtes
- [ ] rspec ./spec/hyku_specs/controllers/admin/groups_controller_spec.rb:43 # Admin::GroupsController as an admin user with an existing group GET #edit 
- [ ] rspec ./spec/hyku_specs/controllers/admin/groups_controller_spec.rb:60 # Admin::GroupsController as an admin user with an existing group GET #remove 
- [ ] rspec ./spec/hyku_specs/controllers/admin/groups_controller_spec.rb:68 # Admin::GroupsController as an admin user with an existing group DELETE #destroy 
- [ ] rspec ./spec/hyku_specs/controllers/admin/groups_controller_spec.rb:25 # Admin::GroupsController as an admin user GET #new 
- [ ] rspec ./spec/hyku_specs/controllers/admin/groups_controller_spec.rb:29 # Admin::GroupsController as an admin user POST #create creates a group when it receives valid attributes
- [ ] rspec ./spec/hyku_specs/controllers/admin/groups_controller_spec.rb:19 # Admin::GroupsController as an admin user GET #index 
- [ ] rspec ./spec/hyku_specs/controllers/admin/groups_controller_spec.rb:18 # Admin::GroupsController as an admin user GET #index 
- [ ] rspec ./spec/hyku_specs/controllers/admin/work_types_controller_spec.rb:25 # Admin::WorkTypesController as an admin user PATCH #update updates attribtes
- [ ] rspec ./spec/hyku_specs/controllers/admin/work_types_controller_spec.rb:19 # Admin::WorkTypesController as an admin user GET #edit 
- [ ] rspec ./spec/hyku_specs/controllers/admin/work_types_controller_spec.rb:18 # Admin::WorkTypesController as an admin user GET #edit 
- [ ] rspec ./spec/hyku_specs/controllers/admin/work_types_controller_spec.rb:8 # Admin::WorkTypesController as an anonymous user GET #index 
- [ ] rspec ./spec/hyku_specs/views/splash/index.html.erb_spec.rb:31 # splash/index.html.erb Admin user with admin_only_tenant_creation=true displays a 'Get Started' button
- [ ] rspec ./spec/hyku_specs/views/splash/index.html.erb_spec.rb:16 # splash/index.html.erb Anonymous or non-Admin user with admin_only_tenant_creation=false displays a 'Get Started' button
- [ ] rspec ./spec/hyku_specs/models/solr_endpoint_spec.rb:6 # SolrEndpoint 
- [ ] rspec ./spec/hyku_specs/features/user_roles_spec.rb:156 # User Roles as a registered user cannot view a users showpage
- [ ] rspec ./spec/hyku_specs/features/user_roles_spec.rb:176 # User Roles as a registered user can view own profile
- [ ] rspec ./spec/hyku_specs/features/user_roles_spec.rb:151 # User Roles as a registered user cannot view users index page
- [ ] rspec ./spec/hyku_specs/features/user_roles_spec.rb:171 # User Roles as a registered user cannot view the manage users page
- [ ] rspec ./spec/hyku_specs/features/user_roles_spec.rb:166 # User Roles as a registered user cannot view the manage groups page
- [ ] rspec ./spec/hyku_specs/features/user_roles_spec.rb:161 # User Roles as a registered user cannot edit a users profile
- [ ] rspec ./spec/hyku_specs/features/user_roles_spec.rb:145 # User Roles as a registered user cannot view Manage Users and Manage Groups in the dashboard sidebar
- [ ] rspec ./spec/hyku_specs/features/user_roles_spec.rb:185 # User Roles as a registered user can edit own profile
- [ ] rspec ./spec/hyku_specs/features/user_roles_spec.rb:120 # User Roles as a user reader can view Manage Users and Manage Groups in the dashboard sidebar
- [ ] rspec ./spec/hyku_specs/features/user_roles_spec.rb:131 # User Roles as a user reader can visit Manage Users and cant delete users
- [ ] rspec ./spec/hyku_specs/features/user_roles_spec.rb:108 # User Roles as a user reader can view a users profile
- [ ] rspec ./spec/hyku_specs/features/user_roles_spec.rb:126 # User Roles as a user reader can visit Manage Users and cant invite users
- [ ] rspec ./spec/hyku_specs/features/user_roles_spec.rb:114 # User Roles as a user reader cannot edit a users profile
- [ ] rspec ./spec/hyku_specs/features/user_roles_spec.rb:97 # User Roles as a user reader can view the users index page
- [ ] rspec ./spec/hyku_specs/features/user_roles_spec.rb:102 # User Roles as a user reader can view a users showpage
- [ ] rspec ./spec/hyku_specs/features/user_roles_spec.rb:203 # User Roles as an unregistered user can sign up
- [ ] rspec ./spec/hyku_specs/features/user_roles_spec.rb:64 # User Roles as a user manager can edit groups name
- [ ] rspec ./spec/hyku_specs/features/user_roles_spec.rb:30 # User Roles as a user manager can visit Manage Users but cannot delete users
- [ ] rspec ./spec/hyku_specs/features/user_roles_spec.rb:75 # User Roles as a user manager can edit groups roles
- [ ] rspec ./spec/hyku_specs/features/user_roles_spec.rb:16 # User Roles as a user manager can view Manage Users and Manage Groups in the dashboard sidebar
- [ ] rspec ./spec/hyku_specs/features/user_roles_spec.rb:51 # User Roles as a user manager can edit a users profile
- [ ] rspec ./spec/hyku_specs/features/user_roles_spec.rb:57 # User Roles as a user manager can view groups and roles
- [ ] rspec ./spec/hyku_specs/features/user_roles_spec.rb:69 # User Roles as a user manager can edit groups users
- [ ] rspec ./spec/hyku_specs/features/user_roles_spec.rb:45 # User Roles as a user manager can visit a users profile showpage
- [ ] rspec ./spec/hyku_specs/features/user_roles_spec.rb:22 # User Roles as a user manager can visit Manage Users and invite users
- [ ] rspec ./spec/hyku_specs/features/user_roles_spec.rb:40 # User Roles as a user manager can visit a users showpage and see the Edit Profile button
- [ ] rspec ./spec/hyku_specs/features/user_roles_spec.rb:35 # User Roles as a user manager can visit the users index page
- [ ] rspec ./spec/hyku_specs/features/user_roles_spec.rb:81 # User Roles as a user manager can remove a group
- [ ] rspec ./spec/features/create_etd_spec.rb:37 # Create a Etd a logged in user with the :work_depositor role can create an Etd
- [ ] rspec ./spec/views/themes/cultural_repository/_user_util_links.html.erb_spec.rb:38 # /themes/cultural_repository/_user_util_links.html.erb when feature flipper is on logged in admin user shows the links
- [ ] rspec ./spec/views/themes/cultural_repository/_user_util_links.html.erb_spec.rb:23 # /themes/cultural_repository/_user_util_links.html.erb when feature flipper is on logged in user shows the links
- [ ] rspec ./spec/views/themes/cultural_repository/_user_util_links.html.erb_spec.rb:53 # /themes/cultural_repository/_user_util_links.html.erb when feature flipper is on logged out user links to login path
- [ ] rspec ./spec/views/themes/cultural_repository/_user_util_links.html.erb_spec.rb:88 # /themes/cultural_repository/_user_util_links.html.erb When feature flipper is off logged in admin user shows the links
- [ ] rspec ./spec/views/themes/cultural_repository/_user_util_links.html.erb_spec.rb:73 # /themes/cultural_repository/_user_util_links.html.erb When feature flipper is off logged in user hides the links
- [ ] rspec ./spec/views/themes/cultural_repository/_user_util_links.html.erb_spec.rb:103 # /themes/cultural_repository/_user_util_links.html.erb When feature flipper is off logged out user hides the login path
- [ ] rspec ./spec/hyku_specs/models/hyrax/file_set_decorator_spec.rb:26 # Hyrax::FileSet converts an AF FileSet to a Valkyrie::FileSet
- [ ] rspec ./spec/jobs/iiif_print/create_relationships_job_decorator_spec.rb:16 # IiifPrint::Jobs::CreateRelationshipsJobDecorator#perform when parent_model is Cdl calls CreateGroupAndAddMembersJob
- [ ] rspec ./spec/hyku_specs/features/splash_spec.rb:15 # The splash page shows a link to login
- [ ] rspec ./spec/hyku_specs/views/labels/edit.html.erb_spec.rb:10 # labels/edit renders the edit site form
- [ ] rspec ./spec/hyku_specs/models/redis_endpoint_spec.rb:4 # RedisEndpoint 
- [ ] rspec ./spec/hyku_specs/controllers/hyrax/images_controller_spec.rb:20 # Hyrax::ImagesController#presenter initializes a presenter
- [ ] rspec ./spec/hyku_specs/controllers/hyrax/images_controller_spec.rb:30 # Hyrax::ImagesController with theming 
- [ ] rspec ./spec/hyku_specs/helpers/application_helper_spec.rb:17 # ApplicationHelper#local_for when term is missing 
- [ ] rspec ./spec/hyku_specs/helpers/application_helper_spec.rb:8 # ApplicationHelper#markdown renders markdown into html
- [ ] rspec ./spec/features/advanced_search_spec.rb:17 # Advanced Search with unauthenticated user can perform advanced search
- [ ] rspec ./spec/features/advanced_search_spec.rb:9 # Advanced Search with unauthenticated user can perform search
- [ ] rspec ./spec/hyku_specs/features/accounts_spec.rb:41 # Accounts administration as an superadmin changes the account service endpoints
- [ ] rspec ./spec/hyku_specs/controllers/admin/group_users_controller_spec.rb:20 # Admin::GroupUsersController as an admin user GET #index 
- [ ] rspec ./spec/hyku_specs/controllers/admin/group_users_controller_spec.rb:21 # Admin::GroupUsersController as an admin user GET #index 
- [ ] rspec ./spec/hyku_specs/controllers/admin/group_users_controller_spec.rb:38 # Admin::GroupUsersController as an admin user modifying group membership DELETE #destroy removes a user from a group when it recieves a group ID
- [ ] rspec ./spec/hyku_specs/controllers/admin/group_users_controller_spec.rb:28 # Admin::GroupUsersController as an admin user modifying group membership POST #create adds a user to a group when it recieves a group ID
- [ ] rspec ./spec/hyku_specs/controllers/admin/group_users_controller_spec.rb:10 # Admin::GroupUsersController as an anonymous user GET #index 
- [ ] rspec ./spec/features/create_oer_spec.rb:41 # Create a Oer a logged in user with the :work_depositor role 
- [ ] rspec ./spec/hyku_specs/helpers/admin_stats_helper_spec.rb:10 # AdminStatsHelper#graph_tag draws html
- [ ] rspec ./spec/hyku_specs/lib/active_job_tenant_spec.rb:33 # ActiveJobTenant.deserialize preserves the original tenant
- [ ] rspec ./spec/presenters/hyrax/oer_presenter_spec.rb:129 # Hyrax::OerPresenter#work_presenters filters out members that are file sets
- [ ] rspec ./spec/presenters/hyrax/oer_presenter_spec.rb:180 # Hyrax::OerPresenter#manifest admin users #editor? 
- [ ] rspec ./spec/presenters/hyrax/oer_presenter_spec.rb:143 # Hyrax::OerPresenter#manifest #sequence_rendering returns a hash containing the rendering information
- [ ] rspec ./spec/presenters/hyrax/oer_presenter_spec.rb:193 # Hyrax::OerPresenter#manifest #manifest_metadata returns an array of metadata values
- [ ] rspec ./spec/presenters/hyrax/oer_presenter_spec.rb:219 # Hyrax::OerPresenter#show_deposit_for? when user does not have depositable collections and user can create a collection returns true
- [ ] rspec ./spec/presenters/hyrax/oer_presenter_spec.rb:229 # Hyrax::OerPresenter#show_deposit_for? when user does not have depositable collections and user can NOT create a collection returns false
- [ ] rspec ./spec/hyku_specs/views/proprietor/accounts/show.html.erb_spec.rb:15 # proprietor/accounts/show renders account admin management form
- [ ] rspec ./spec/views/themes/institutional_repository/_user_util_links.html.erb_spec.rb:82 # /themes/institutional_repository/_user_util_links.html.erb When feature flipper is off logged in admin user shows the links
- [ ] rspec ./spec/views/themes/institutional_repository/_user_util_links.html.erb_spec.rb:97 # /themes/institutional_repository/_user_util_links.html.erb When feature flipper is off logged out user hides the login path
- [ ] rspec ./spec/views/themes/institutional_repository/_user_util_links.html.erb_spec.rb:67 # /themes/institutional_repository/_user_util_links.html.erb When feature flipper is off logged in user hides the links
- [ ] rspec ./spec/views/themes/institutional_repository/_user_util_links.html.erb_spec.rb:50 # /themes/institutional_repository/_user_util_links.html.erb when feature flipper is on logged out user links to login path
- [ ] rspec ./spec/views/themes/institutional_repository/_user_util_links.html.erb_spec.rb:35 # /themes/institutional_repository/_user_util_links.html.erb when feature flipper is on logged in admin user shows the links
- [ ] rspec ./spec/views/themes/institutional_repository/_user_util_links.html.erb_spec.rb:20 # /themes/institutional_repository/_user_util_links.html.erb when feature flipper is on logged in user shows the links
- [ ] rspec ./spec/models/etd_spec.rb:136 # Etd metadata properties :resource_type is a property
- [ ] rspec ./spec/models/etd_spec.rb:37 # Etd metadata properties :creator is a property
- [ ] rspec ./spec/models/etd_spec.rb:118 # Etd metadata properties :identifier is a property
- [ ] rspec ./spec/models/etd_spec.rb:46 # Etd metadata properties :keyword is a property
- [ ] rspec ./spec/models/etd_spec.rb:145 # Etd metadata properties :format is a property
- [ ] rspec ./spec/models/etd_spec.rb:73 # Etd metadata properties :description is a property
- [ ] rspec ./spec/models/etd_spec.rb:109 # Etd metadata properties :language is a property
- [ ] rspec ./spec/models/etd_spec.rb:28 # Etd metadata properties :title is a property
- [ ] rspec ./spec/models/etd_spec.rb:55 # Etd metadata properties :rights_statement is a property
- [ ] rspec ./spec/models/etd_spec.rb:208 # Etd metadata properties :department is a property
- [ ] rspec ./spec/models/etd_spec.rb:172 # Etd metadata properties :degree_discipline is a property
- [ ] rspec ./spec/models/etd_spec.rb:64 # Etd metadata properties :contributor is a property
- [ ] rspec ./spec/models/etd_spec.rb:127 # Etd metadata properties :source is a property
- [ ] rspec ./spec/models/etd_spec.rb:163 # Etd metadata properties :degree_level is a property
- [ ] rspec ./spec/models/etd_spec.rb:100 # Etd metadata properties :subject is a property
- [ ] rspec ./spec/models/etd_spec.rb:199 # Etd metadata properties :committee_member is a property
- [ ] rspec ./spec/models/etd_spec.rb:91 # Etd metadata properties :publisher is a property
- [ ] rspec ./spec/models/etd_spec.rb:181 # Etd metadata properties :degree_grantor is a property
- [ ] rspec ./spec/models/etd_spec.rb:82 # Etd metadata properties :license is a property
- [ ] rspec ./spec/models/etd_spec.rb:154 # Etd metadata properties :degree_name is a property
- [ ] rspec ./spec/models/etd_spec.rb:190 # Etd metadata properties :advisor is a property
- [ ] rspec ./spec/hyku_specs/indexers/generic_work_resource_indexer_spec.rb:12 # GenericWorkResourceIndexer behaves like a Hyrax::Resource indexer #to_solr indexes base resource fields
- [ ] rspec ./spec/hyku_specs/features/featured_collections_spec.rb:16 # The homepage shows featured collections
- [ ] rspec ./spec/hyku_specs/features/featured_collections_spec.rb:26 # The homepage as an admin shows featured collections that I can sort
- [ ] rspec ./spec/models/generic_work_spec.rb:14 # GenericWork 
- [ ] rspec ./spec/hyku_specs/views/proprietor/users/new.html.erb_spec.rb:34 # proprietor/users/new renders new proprietor_user form
- [ ] rspec ./spec/hyku_specs/features/proprietor_spec.rb:19 # Proprietor administration as an superadmin has a navbar link to an account admin section
- [ ] rspec ./spec/hyku_specs/features/proprietor_spec.rb:25 # Proprietor administration as an superadmin has a navbar link to logout
- [ ] rspec ./spec/hyku_specs/models/fcrepo_endpoint_spec.rb:7 # FcrepoEndpoint 
- [ ] rspec ./spec/omni_auth/strategies/open_id_connect_decorator_spec.rb:84 # OmniAuth::Strategies::OpenIDConnect#options method source_location is expected to contain exactly "/app/samvera/hyrax-webapp/lib/omni_auth/strategies/open_id_connect_decorator.rb" and Integer
- [ ] rspec ./spec/features/splash_spec.rb:15 # The splash page shows the page, displaying the Hyku for Consortia content
- [ ] rspec ./spec/hyku_specs/views/_user_util_links.html.erb_spec.rb:82 # /_user_util_links.html.erb When feature flipper is off logged in admin user shows the links
- [ ] rspec ./spec/hyku_specs/views/_user_util_links.html.erb_spec.rb:67 # /_user_util_links.html.erb When feature flipper is off logged in user hides the links
- [ ] rspec ./spec/hyku_specs/views/_user_util_links.html.erb_spec.rb:97 # /_user_util_links.html.erb When feature flipper is off logged out user hides the login path
- [ ] rspec ./spec/hyku_specs/views/_user_util_links.html.erb_spec.rb:35 # /_user_util_links.html.erb when feature flipper is on logged in admin user shows the links
- [ ] rspec ./spec/hyku_specs/views/_user_util_links.html.erb_spec.rb:50 # /_user_util_links.html.erb when feature flipper is on logged out user links to login path
- [ ] rspec ./spec/hyku_specs/views/_user_util_links.html.erb_spec.rb:20 # /_user_util_links.html.erb when feature flipper is on logged in user shows the links
- [ ] rspec ./spec/hyku_specs/features/labels_spec.rb:55 # Site labels configuration as an administrator institution name full updates the full institution name in the agreement text
- [ ] rspec ./spec/hyku_specs/features/labels_spec.rb:37 # Site labels configuration as an administrator institution name updates the institution name in the agreement text
- [ ] rspec ./spec/hyku_specs/features/labels_spec.rb:20 # Site labels configuration as an administrator application name updates the application name in the <head> <title>
- [ ] rspec ./spec/hyku_specs/features/labels_spec.rb:13 # Site labels configuration as an administrator application name updates the application name in the brand bar
- [ ] rspec ./spec/actors/hyrax/actors/oer_actor_spec.rb:32 # Hyrax::Actors::OerActor#create adds a related version
- [ ] rspec ./spec/actors/hyrax/actors/oer_actor_spec.rb:54 # Hyrax::Actors::OerActor#update updates the related item
- [ ] rspec ./spec/actors/hyrax/actors/oer_actor_spec.rb:61 # Hyrax::Actors::OerActor#update removes the related version items
- [ ] rspec ./spec/actors/hyrax/actors/oer_actor_spec.rb:80 # Hyrax::Actors::OerActor#update adds and removes relationships on same submit
- [ ] rspec ./spec/actors/hyrax/actors/oer_actor_spec.rb:39 # Hyrax::Actors::OerActor#update updates the previous version items
- [ ] rspec ./spec/actors/hyrax/actors/oer_actor_spec.rb:44 # Hyrax::Actors::OerActor#update updates the newer version items
- [ ] rspec ./spec/actors/hyrax/actors/oer_actor_spec.rb:49 # Hyrax::Actors::OerActor#update updates the alternate version
- [ ] rspec ./spec/hyku_specs/helpers/shared_search_helper_spec.rb:45 # SharedSearchHelper shared search records in development returns #generate_work_url with a query
- [ ] rspec ./spec/hyku_specs/helpers/shared_search_helper_spec.rb:40 # SharedSearchHelper shared search records in development returns #generate_work_url
- [ ] rspec ./spec/hyku_specs/helpers/shared_search_helper_spec.rb:24 # SharedSearchHelper shared search records in production returns #generate_work_url
- [ ] rspec ./spec/hyku_specs/helpers/shared_search_helper_spec.rb:29 # SharedSearchHelper shared search records in production returns #generate_work_url with a query
- [ ] rspec ./spec/hyku_specs/features/work_editor_role_spec.rb:151 # Work Editor role destroy permissions cannot see the delete button on the work show page
- [ ] rspec ./spec/hyku_specs/features/work_editor_role_spec.rb:83 # Work Editor role create permissions can create a work
- [ ] rspec ./spec/hyku_specs/features/work_editor_role_spec.rb:124 # Work Editor role edit permissions can see the edit button on the work show page
- [ ] rspec './spec/hyku_specs/features/work_editor_role_spec.rb[1:1:3:1]' # Work Editor role read permissions with restricted visibility_setting can see the work in search results
- [ ] rspec './spec/hyku_specs/features/work_editor_role_spec.rb[1:1:1:1]' # Work Editor role read permissions with open visibility_setting can see the work in search results
- [ ] rspec './spec/hyku_specs/features/work_editor_role_spec.rb[1:1:2:1]' # Work Editor role read permissions with authenticated visibility_setting can see the work in search results
- [ ] rspec ./spec/hyku_specs/views/proprietor/accounts/new.html.erb_spec.rb:10 # proprietor/accounts/new renders new account form
- [ ] rspec ./spec/hyku_specs/features/work_approval_permissions_spec.rb:93 # Work approval permissions when signed in as a user with no special access cannot see the workflow actions widget on the work's show page
- [ ] rspec ./spec/hyku_specs/features/work_approval_permissions_spec.rb:76 # Work approval permissions when signed in as a work depositor cannot see the workflow actions widget on the work's show page
- [ ] rspec ./spec/hyku_specs/features/work_approval_permissions_spec.rb:30 # Work approval permissions when signed in as an admin can see the workflow actions widget on the work's show page
- [ ] rspec ./spec/hyku_specs/features/work_approval_permissions_spec.rb:53 # Work approval permissions when signed in as a work editor can see the workflow actions widget on the work's show page
- [ ] rspec ./spec/hyku_specs/views/proprietor/users/edit.html.erb_spec.rb:35 # proprietor/users/edit renders the edit proprietor_user form
- [ ] rspec ./spec/hyku_specs/controllers/hyrax/generic_works_controller_spec.rb:22 # Hyrax::GenericWorksController#presenter initializes a presenter

Randomized with seed 21991

Coverage report generated for RSpec to /app/samvera/coverage. 6337 / 8232 LOC (76.98%) covered.
